config/params.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+use Psr\Log\LogLevel;
+
+return [
+    'aliases' => [
+        '@root' => dirname(__DIR__),
```

Comment:
Let's do it according to https://trello.com/c/myCLrBMO/67-standardize-base-set-of-aliases